tcms/testcases/tests.py

```diff
@@ -197,3 +198,33 @@ def test_remove_tag(self):
 
         tag_pks = list(self.case.tag.all().values_list('pk', flat=True))
         self.assertEqual([self.tag_fedora.pk], tag_pks)
+
+
+class CaseTagFormTest(BasePlanCase):
```

Comment:
Looks this test case does not need inherit from ``BasePlanCase``, as no data created in super ``setUpTestData`` is used.